arch/arm/src/nrf52/nrf52_rtc.h

```diff
@@ -74,6 +76,18 @@ enum nrf52_rtc_irq_e
   NRF52_RTC_INT_COMPARE3 = 5,
 };
 
+/* RTC Events */
```

Comment:
Don't you think we should combine this with 'enum nrf52_rtc_irq_e' defined above ?
They carry the same information and there is no point in duplicating them.

Reply:
If I do that I risk breaking code which used those definitions. Not sure what is preferable.

Reply:
If that breaks someone's compilation, fixing it is trivial.
I think the name 'nrf52_rtc_evt_e' seems more appropriate.

Reply:
So you suggest removing the irq enum, leave the evt one and use that, right?

Reply:
> So you suggest removing the irq enum, leave the evt one and use that, right?

Exactly.

Reply:
done